scripts/grest-helper-scripts/setup-grest.sh

```diff
@@ -163,7 +163,8 @@
   kill_running_cron_jobs() {
     # Currently just killing any cron entry other than the first one
     # There is probably a better (more precise) way of doing this
-    kill -9 $(pgrep cron | awk 'FNR > 1 {print $1}')
+    echo "Stopping currently running cron jobs..."
+    sudo kill -9 $(pgrep cron | awk 'FNR > 1 {print $1}')
```

Comment:
Would it be safer to only kill entries based on path of instance? 
Something like:
`lsof | grep "${CRON_SCRIPTS_DIR}" | awk '{print $2}' | sudo xargs kill -9`